main.tf

```diff
@@ -5,11 +5,36 @@ provider "aws" {
 resource "aws_s3_bucket" "demo-s3" {
   bucket = "${var.s3_bucket_prefix}-terraformcloud"
   versioning {
-    enabled = false
-    mfa_delete = false
+    enabled    = true
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure versioning is enabled for AWS S3 Buckets**
 
**Vulnerability**
Enabling S3 versioning will enable easy recovery from both unintended user actions, like deletes and overwrites.
 
**Remediation**
In AWS Console - 
 1. Sign in to the AWS Console and go to the Amazon S3 console.
 2. In the Buckets list, selec the bucket.
 3. Select Properties.
 4. Under Bucket Versioning, select Edit.
 5. Select Suspend or Enable, and then select Save changes.
 In Terraform - 
 Set the 'versioning.enabled' attribute to 'true'.

source: accurics-devenv